Build/AzureDevOps/azure-pipelines.yml

```diff
@@ -9,9 +9,32 @@ jobs:
     pool:
       vmImage: 'VS2017-Win2016'
 
-    variables:
-      buildPlatform: 'Win32'
-      buildConfiguration: 'Release'
+    strategy:
+      matrix:
+        win32-release:
+          buildPlatform: win32
+          buildConfiguration: release
```

Comment:
This should be "Release" or "Debug".

The configuration in this context is generated by cmake and will be the string "Debug" or "Release".  This might work, because the Windows filesystem is not case sensitive, but we should try to use the correct name if possible (look at the folders after a build to see the file case).

Reply:
Ah, you're right - good catch! I'm just going to defer to CMake's defaults for the platform and configuration casing everywhere. (So e.g. back to Win32/Release, but make the folder name windows_Win32).

Reply:
That doesn't seem better.  What about adding a "buildPath" variable to the matrix and set that to cmake_win32 instead of generating the path.  Way more flexible that way if anything ever breaks and very easy to read.